test/unit/test_sampler.py

```diff
@@ -98,32 +98,32 @@ def test_unsupported_values_for_sampler_options(self, opt):
 
     def test_run_default_options(self):
         """Test run using default options."""
-        session = MagicMock(spec=MockSession)
+        session = MagicMock(spec=MockSession, _backend="common_backend")
         options_vars = [
             (
-                SamplerOptions(dynamical_decoupling="XX"),  # pylint: disable=unexpected-keyword-arg
-                {"dynamical_decoupling": "XX"},
+                SamplerOptions(  # pylint: disable=unexpected-keyword-arg
+                    dynamical_decoupling={"sequence_type": "XX"}
+                ),
+                {"dynamical_decoupling": {"sequence_type": "XX"}},
             ),
             (
-                SamplerOptions(optimization_level=3),  # pylint: disable=unexpected-keyword-arg
-                {"transpilation": {"optimization_level": 3}},
+                SamplerOptions(default_shots=1000),  # pylint: disable=unexpected-keyword-arg
+                {"default_shots": 1000},
```

Comment:
The pylint disabling is necessary? `dynamical_decoupling` and `default_shots` are valid options.

Reply:
I believe there's an incompatibility between pylint and pydantic causing this lint error.